app/bundles/PageBundle/Model/PageModel.php

```diff
@@ -770,14 +726,16 @@ public function processPageHit(Hit $hit, $page, Request $request, Lead $lead, $t
      */
     public function getHitQuery(Request $request, $page = null)
     {
-        $get  = $request->query->all();
```

Comment:
Are you sure this can be removed? I'm not sure if it was there for some use case. Would it hurt to leave it as it was?

Reply:
it is still there, it is on line 730

Reply:
I can see `$query = $request->request->all();` on line 730. I'm talking about `$get  = $request->query->all();`. One is for POST, the other for GET.

Reply:
yes, 730 is for POST. Then on line 737 I get the GET parameters from the url. If I understood the code correct than there are two ways: direct or via javascript api. In this new way I also catch the GET parameters that are passed in via page_url.

Reply:
@escopecz what are your thoughts?

Reply:
Sounds good. Sorry for late reply. I was hoping that an Acquia engineer on community rotation will finish the job. As usual, there were different priorities.